src/rest-client.ts

```diff
@@ -674,11 +703,11 @@ export class RestClient extends BaseRestClient {
     });
   }
 
-  getMaxWithdrawals(ccy?: string): Promise<any[]> {
+  getMaxWithdrawals(ccy?: string): Promise<MaxWithdrawals[]> {
```

Comment:
```suggestion
  getMaxWithdrawals(ccy?: string): Promise<MaxWithdrawal[]> {
```

Missed this one - sorry to be picky but it's the last one. Should be singular. It's an array of "MaxWithdrawal", each element in the array represents one `MaxWithdrawal` object.